### ISSUE TITLE
Quickreply bug

### ISSUE THREAD

**Issue by sargijeh** [wasdennnoch/AndroidN-ify#1013]
Quickreply button doesn't to anything when I press it on Headsups!
But on notification bar (qs page) it is okay

**Comment by Maxr1998** [comment]
Which app? This should be fixed since ages.

**Comment by sargijeh** [comment]
All the apps including messaging, whatsapp, plus messenger.
Also the headsup's are not fullwidth, isn't it the cause? @maxr1998
Xperia LP stock

**Comment by Maxr1998** [comment]
Ah, LP. Might explain it. I remember that Lollipop handled Heads up different, so I couldn't add the quickreply to them.

**Comment by Jrdnkasparek** [comment]
Can confirmed this happening as well for me

Messenger Beta by Klinker

There is also two reply buttons showing up as well. Klinker says it is xposed module causing this

![16 - 1](https://cloud.githubusercontent.com/assets/20908630/19399493/65bb14ca-9217-11e6-8d87-f61cacccc51f.png)

**Comment by Maxr1998** [comment]
Yes, it is. I only add the quick reply item extracted from Android Wear or Auto to the notification (the first item), but don't remove the others, to prevent removing other useful ones like the call item.
You could ask the dev to add a RemoteInput object to the call button, then I can replace the button and don't have to add a new one. Or you just ask him to contact me :)

**Comment by Jrdnkasparek** [comment]
I will give him a heads up and let you know

**Comment by sargijeh** [comment]
Actually no one really answer my question even on xda, please tell me what is wrong with the lollipop and how can I make it's headsupfull width,
What is the problem and why there is no explanation for it even in xda? 
Btw, I love your maxlock module😄
 @maxr1998

**Comment by Maxr1998** [comment]
I don't know what's the issue with Xperia's code, sadly I also can't debug it..
Thanks :)

**Comment by Maxr1998** [comment]
Yes, it is.

**Comment by Jrdnkasparek** [comment]
I think there is possible confusion that Klinker is not understanding, he said: "Adding a remote input to the call button makes no sense... The call button doesn't get any input, it just sends out an action. Remote inputs are used for buttons like the reply button, where you can respond from your watch, or from the direct reply on android N."

**Comment by Jrdnkasparek** [comment]
but the point was there are two reply buttons, one uses direct reply and the other opens up his apps quick reply
